prestoadmin/util/slider.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Utility functions for dealing with slider
+"""
+
+
+def degarbage_json(garbage_output):
+    nest_level = 0
```

Comment:
1. Guess who was a Grumpy Gus when he was writing this :-) I'll change it. Slider doesn't separate the JSON from its messages into stdout and stderr.
2. That would be simpler, but it doesn't guarantee that you end up with a balanced bunch of braces. Mind you, this isn't doing anything else to ensure that it returns well-formed JSON...